### PR TITLE
Update tutorial.rst

### PR DESCRIPTION
**Pull request by Mubarraqqq**
Hi @joshmoore @MSanKeys963. 
I made an update on the Zarr Tutorial and also included a new compressor.  

TODO:
* [ ] Add unit tests and/or doctests in docstrings
* [ ] Add docstrings and API docs for any new/modified user-facing classes and functions
* [ ] New/modified features documented in docs/tutorial.rst
* [ ] Changes documented in docs/release.rst
* [ ] GitHub Actions have all passed
* [ ] Test coverage is 100% (Codecov passes)

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
...
+    Shape              : (10000, 10000)
+    Chunk shape        : (1000, 1000)
+    Order              : C
+    Read-only          : False
+    Compressor         : Zstd(level=1)
+    Store type         : zarr.storage.KVStore
+    No. bytes          : 400000000 (381.5M)
+    No. bytes stored   : 299348530 (285.5M)
+    Storage ratio      : 1.3
+    Chunks initialized : 100/100
```
Comment (MSanKeys963) [inline docs/tutorial.rst:221]: I think this section should be in a [code block](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

Comment (MSanKeys963) [inline docs/tutorial.rst:221]: This section should be in a [code block](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

File: docs/tutorial.rst [hunk 2]
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
...
+
+
+Here is also an array using GZip compression, level 2::
+
+    >>> from numcodecs import GZip
+    >>> z = zarr.array(np.arange(100000000, dtype='i4').reshape(10000, 10000),
+    ...                chunks=(1000, 1000), compressor=GZip(level=2))
+    >>> z.compressor
+    GZip(level=2)
```
Comment (MSanKeys963) [inline docs/tutorial.rst:230]: This too.

File: docs/tutorial.rst [hunk 3]
```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
...
+    Shape              : (10000, 10000)
+    Chunk shape        : (1000, 1000)
+    Order              : C
+    Read-only          : False
+    Compressor         : GZip(level=2)
+    Store type         : zarr.storage.KVStore
+    No. bytes          : 400000000 (381.5M)
+    No. bytes stored   : 138601771 (132.2M)
+    Storage ratio      : 2.9
+    Chunks initialized : 100/100
```
Comment (MSanKeys963) [inline docs/tutorial.rst:246]: This as well.

File: docs/tutorial.rst (hunk 2 above)
Comment (MSanKeys963) [inline docs/tutorial.rst:230]: This too.

Comment (Mubarraqqq) [reply docs/tutorial.rst:230]: Sorry that was my mistake @MSanKeys963.
I would work on that right away!

Comment (Mubarraqqq) [reply docs/tutorial.rst:230]: Hello @MSanKeys963 . I have resolved the issue

**Comment by MSanKeys963** [comment]
Hi @Mubarraqqq. I've added some minor changes here: https://github.com/Mubarraqqq/zarr-python/pull/1
Please have a look.

**Comment by Mubarraqqq** [comment]
> Hi @Mubarraqqq. I've added some minor changes here: [Mubarraqqq#1](https://github.com/Mubarraqqq/zarr-python/pull/1) Please have a look.

Okay, I will have a look. 

> Hi @Mubarraqqq. I've added some minor changes here: [Mubarraqqq#1](https://github.com/Mubarraqqq/zarr-python/pull/1) Please have a look.

I've checked it out and merged it @MSanKeys963

**Comment by jhamman** [comment]
This has unfortunately gone stale. Will it be revived or should we close this?

**Comment by jhamman** [comment]
Closing as stale. Feel free to reopen if you want to revive the effort.

**Pull request closed without merging**